apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.appup.src

```diff
@@ -1,24 +1,20 @@
-%% -*-: erlang -*-
-
-{"4.3.3",
-  [
-    {<<"4.3.[1-2]">>, [
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {"4.3.0", [
-      {load_module, emqx_bridge_worker, brutal_purge, soft_purge, []},
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {<<".*">>, []}
-  ],
-  [
-    {<<"4.3.[1-2]">>, [
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {"4.3.0", [
-      {load_module, emqx_bridge_worker, brutal_purge, soft_purge, []},
-      {load_module, emqx_bridge_mqtt_actions, brutal_purge, soft_purge, []}
-    ]},
-    {<<".*">>, []}
-  ]
-}.
+%% -*- mode: erlang -*-
+{VSN,
```

Comment:
Here we should put the new version value.

Reply:
this can actually work.